traveler-ios-ui/TravelerKitUI/ViewControllers/BookableConfirmationViewController.swift

```diff
@@ -14,7 +14,7 @@ protocol BookingConfirmationViewControllerDelegate: class {
 }
 
 class BookingConfirmationViewController: UIViewController {
-    var product: Product?
+    var product: BookingItemDetails?
```

Comment:
I shouldn't need the details to do this part should I?

Reply:
See reply above